{{cookiecutter.project_slug}}/Makefile

```diff
@@ -340,56 +332,57 @@ kill-train:  ### Terminate the training job (set up env var 'RUN' to specify the
-			RESULTS_DIR_STORAGE=$(RESULTS_DIR_STORAGE)/sweep-${SWEEP_REAL} \
-			_SWEEP_DESCRIPTION_POSTFIX=":${SWEEP_REAL}" \
-			_EXTRA_VOLUME_OPTION="--volume $(PROJECT_PATH_STORAGE)/wandb:$(PROJECT_PATH_ENV)/wandb:rw" \
-			TRAIN_STREAM_LOGS=no ; \
-	done
-	@echo -e "\nStarted ${N_HYPERPARAM_JOBS} hyper-parameter search jobs of sweep ${SWEEP_REAL}.\nUse 'neuro ps' and 'neuro status <job>' to check."
-
-.PHONY: kill-hypertrain
-kill-hypertrain:  ### Terminate hyper-parameter search training jobs of the latest sweep (set up env var 'SWEEP' to specify the sweep)
-	@jobs=$$(neuro --quiet ps --description="$(PROJECT_ID):train:$(SWEEP_REAL)") && \
-	echo "Killing jobs of the sweep ${SWEEP_REAL}..." && \
-	$(NEURO) kill $${jobs:-placeholder} || :
+	sweep=$$(tail -1 $(WANDB_SWEEPS_FILE)) && \
+	$(NEURO) mkdir -p $(PROJECT_PATH_STORAGE)/$(RESULTS_DIR)/sweep-$$sweep && \
+	echo "Running ${N_HYPERPARAM_JOBS} jobs of sweep '$$sweep'..." && \
+	for index in $$(seq 1 $(N_HYPERPARAM_JOBS)) ; do \
+		echo -e "\nStarting job $$index..." ; \
+		$(NEURO) run \
+			--name $(TRAIN_JOB)-$$sweep-$$index \
+			--description "$(PROJECT_ID):train:$$sweep" \
+			--preset $(PRESET) \
+			--detach \
+			--volume $(DATA_DIR_STORAGE):$(PROJECT_PATH_ENV)/$(DATA_DIR):ro \
+			--volume $(PROJECT_PATH_STORAGE)/$(CODE_DIR):$(PROJECT_PATH_ENV)/$(CODE_DIR):ro \
+			--volume $(PROJECT_PATH_STORAGE)/$(CONFIG_DIR):$(PROJECT_PATH_ENV)/$(CONFIG_DIR):ro \
+			--volume $(PROJECT_PATH_STORAGE)/sweep-$$sweep:$(PROJECT_PATH_ENV)/sweep-$$sweep:rw \
+			--volume $(PROJECT_PATH_STORAGE)/wandb:$(PROJECT_PATH_ENV)/wandb:rw \
+			--env PYTHONPATH=$(PROJECT_PATH_ENV) \
+			--env EXPOSE_SSH=yes \
+			--env JOB_TIMEOUT=0 \
+			${OPTION_GCP_CREDENTIALS} ${OPTION_AWS_CREDENTIALS} ${OPTION_WANDB_CREDENTIALS} \
+			$(CUSTOM_ENV_NAME) \
+			bash -c "cd $(PROJECT_PATH_ENV) && wandb status && wandb agent $$sweep"; \
+	done; \
+	echo -e "\nStarted $(N_HYPERPARAM_JOBS) hyper-parameter search jobs of sweep '$$sweep'.\nUse 'neuro ps' and 'neuro status <job>' to check."
 
 .PHONY: kill-hypertrain-all
 kill-hypertrain-all:  ### Terminate all hyper-parameter search training jobs of all the sweeps
-	@[ ! -f ${WANDB_SWEEPS_FILE} ] || \
-	for sweep in $$(tac ${WANDB_SWEEPS_FILE} || tail -r ${WANDB_SWEEPS_FILE}); do \
-		jobs=$$(neuro --quiet ps --description="$(PROJECT_ID):train:$${sweep}") && \
-		echo "Killing jobs of the sweep $${sweep}..." && \
-		$(NEURO) kill $${jobs:-placeholder} ; \
-	done ; \
-	echo "Please remove unused sweeps from the file ${WANDB_SWEEPS_FILE}" || :
+	@[ ! -f "$(WANDB_SWEEPS_FILE)" ] && echo "File '$(WANDB_SWEEPS_FILE)' does not exist. Abort." || { \
+		for sweep in $$(tac $(WANDB_SWEEPS_FILE)); do \
+			jobs=$$(neuro --quiet ps --description="$(PROJECT_ID):train:$$sweep") && \
+			echo "Killing jobs of the sweep '$$sweep'..." && \
+			$(NEURO) kill $${jobs:-placeholder} ||: ; \
+		done; \
+		echo "Please remove unused sweeps from the file '$(WANDB_SWEEPS_FILE)'." ; \
+	}
```

Comment:
1. added a message indicating that the file with sweeps does not exist.
2. removed unnecessary part `|| tail -r ${WANDB_SWEEPS_FILE}`